addons/api/addon/models/target.js

```diff
@@ -15,6 +16,10 @@ export default class TargetModel extends GeneratedTargetModel {
   // =services
 
   @service store;
+  @service ipc;
+  @service router;
+  @service session;
+  @service confirm;
```

Comment:
Same with these, we're tightly coupling API to this desktop implementation unfortunately